### ISSUE TITLE
Orphaned logging page?

### ISSUE THREAD

**Issue by michaelstingl** [owncloud/docs#3591]
## WHAT Needs to be Documented?

I found this with search:
https://doc.owncloud.com/server/admin_manual/configuration/server/occ_command/logging_commands.html
(but it's not part of the navigation)

## WHERE Does This Need To Be Documented (Link)?

?

## WHY Should This Change Be Made?

Every page should have proper navigation.

## (Optional) What Type Of Content Change Is This? 
- [ ] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [x] Bug Fix to Existing Content

## (Optional) Which Manual Does This Relate To?
- [x] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [ ] Desktop Client
- [ ] Other

**Comment by voroyam** [comment]
My answer was regarding something I wanted to remove from the docs (or the branch of the docs) that are not maintained anymore, like 10.0 or 10.1.

The reason is that if you try to remove that - the build process will fail (if you want you can go back to the PR and check the drone logs). And to fix that you need manpower and try to convince someone that you want to fix something that is dead anyway. You won't get far.

This new find however is something else. I also have found several of those things, I think they are snippets that are included in a larger document, thus not required to be in the navigation bar on the left. Like if you go to the manual installation guide from @mmattel (the large one) there are a lot of those. Includes that navigate you to a different page that is not in the nav.adoc page.

My advice would be to always mention the NEW doc Team, @mmattel and @EParzefall when you have questions about the docs. They are in charge now.

**Comment by voroyam** [comment]
<img width="1277" alt="image" src="https://user-images.githubusercontent.com/24212810/119697551-95834000-be50-11eb-8ba3-32788eb2e0bd.png">

**Comment by voroyam** [comment]
Yeah, you have found the includes to the occ document. This was a change directed by Matthew and Martin, to split up the occ command document in to these kinds of includes.

**Comment by voroyam** [comment]
This is how the page looks included:

https://doc.owncloud.com/server/10.7/admin_manual/configuration/server/occ_command.html#logging-commands

**Comment by michaelstingl** [comment]
> to split up the occ command document in to these kinds of includes.

Ah, I get the idea. Then we better exclude the includes from the search index…

**Comment by mmattel** [comment]
@voroyam is right, not each document is necessarily part of the navigation (but each document must be referenced bei either an xref or an include, else it is orphaned. We have to differenciate between referencing (xref) or including (include) a document. While xref just references a document, include does what it says and embeds a document into a page. Includes are a necessary procedure to keep big documents maintainable, like we have with the occ command set.

Building docs renders html files from all adoc files. When adoc files contain includes, the final html for this page contains all includes. This is what we see and by design not changable.

This now explains why indexing for ES shows pages, which are the origin of an include. The indexing logic behind is located at https://github.com/owncloud/docs/blob/901afc5f065c428023aad633f6f6c2d11dfb9aba/generator/generate-site.js#L38
and can be adopted - but there will be some effort behind to find the correct logic.

One idea may be post creating the htmls files, to create a map of indexable files (all unique files which are linked somewhere) and use them for ES. 

@EParzefall FYI

**Comment by DeepDiver1975** [comment]
🤖 Automated issue triage by Claude Code review agent.

This issue (orphaned `logging_commands.html` page, and included snippets leaking into the search index) is **resolved** by the documentation toolchain overhaul that has happened since 2021, so we're closing it as obsolete.

What changed:

1. **The orphaned pages are gone.** The URLs that triggered this report now 404:
   - `…/occ_command/logging_commands.html` → 404
   - `…/occ_command/integrity_check_commands.html` → 404 (the example @cortho added)

   They no longer exist as standalone pages on the site.

2. **Included snippets are now excluded from the search index by design.** The occ command set is still split into includes (this keeps the big document maintainable, as @mmattel and @voroyam described), but those snippets are now stored as **Antora partials with a leading underscore**, e.g.
   `modules/admin_manual/pages/configuration/server/occ_commands/core_commands/_logging_commands.adoc`
   and included into `occ_command.adoc` via `include::…/_logging_commands.adoc[]`.

   Antora does **not** publish underscore-prefixed page files as standalone HTML, so these snippets never land in the built `public/` output — and therefore can never be picked up by the search indexer. The exact behavior requested here ("exclude such included snippets from the search index") is now structurally guaranteed.

3. The original implementation premise is also obsolete: the custom Elasticsearch indexing logic in `generator/generate-site.js` referenced in the thread no longer exists. The docs build moved to **Antora 3.x with Pagefind** for search.

If you ever spot an individual snippet page still showing up in search on a current release, please reopen with the offending URL and we'll take a look. Thanks @michaelstingl, @cortho, @voroyam, and @mmattel for the original investigation\!
